okapi-ir/src/test/scala/org/opencypher/okapi/ir/impl/typer/SchemaTyperTest.scala

```diff
@@ -490,126 +472,122 @@ class SchemaTyperTest extends BaseTestSuite with Neo4jAstTestSupport with Mockit
   }
 
   it("typing of list indexing") {
-    implicit val context: TypeTracker = TypeTracker.empty.updated(Parameter("param", symbols.CTAny)(pos), CTInteger)
-
-    assertExpr.from("[1, 2][15]") shouldHaveInferredType CTInteger.nullable
-    assertExpr.from("[3.14, -1, 5000][15]") shouldHaveInferredType CTNumber.nullable
-    assertExpr.from("[[], 1, true][15]") shouldHaveInferredType CTAny.nullable
-    assertExpr.from("[1, 2][1]") shouldHaveInferredType CTInteger.nullable
-    assertExpr.from("[3.14, -1, 5000][$param]")(TypeTracker.empty.withParameters(Map("param" -> CypherValue(42L)))) shouldHaveInferredType CTNumber.nullable
-    assertExpr.from("[[], 1, true][$param]")(TypeTracker.empty.withParameters(Map("param" -> CypherValue(21L)))) shouldHaveInferredType CTAny.nullable
+    assertExpr.from("LIST_INTEGER[15]") shouldHaveInferredType CTInteger.nullable
+    assertExpr.from("LIST_NUMBER[15]") shouldHaveInferredType CTNumber.nullable
+    assertExpr.from("LIST_ANY[15]") shouldHaveInferredType CTAny.nullable
+    assertExpr.from("LIST_INTEGER[1]") shouldHaveInferredType CTInteger.nullable
+    assertExpr.from("LIST_NUMBER[$INTEGER_0]") shouldHaveInferredType CTNumber.nullable
+    assertExpr.from("LIST_ANY[$INTEGER_0]") shouldHaveInferredType CTAny.nullable
   }
 
   it("typing of map indexing") {
-    implicit val context: TypeTracker = TypeTracker.empty
-      .updated(Variable("map")(pos), CTMap(Map("foo" -> CTInteger, "bar" -> CTBoolean)))
-      .updated(Variable("stringKey")(pos), CTString)
-      .updated(Variable("intKey")(pos), CTInteger)
-      .withParameters(Map(
-        "stringParam" -> CypherValue("bar"),
-        "intParam" -> CypherValue(42L)
-      ))
+    //implicit val context: TypeTracker = TypeTracker.empty
+    //  .updated(Variable("map")(pos), CTMap(Map("foo" -> CTInteger, "bar" -> CTBoolean)))
+    //  .updated(Variable("stringKey")(pos), CTString)
+    //  .updated(Variable("intKey")(pos), CTInteger)
+    //  .withParameters(Map(
+    //    "stringParam" -> CypherValue("bar"),
+    //    "intParam" -> CypherValue(42L)
+    //  ))
 
-    assertExpr.from("""map["foo"]""") shouldHaveInferredType CTInteger
-    assertExpr.from("""map["bar"]""") shouldHaveInferredType CTBoolean
-    assertExpr.from("""map["baz"]""") shouldHaveInferredType CTVoid
+    assertExpr.from("""MAP["INTEGER"]""") shouldHaveInferredType CTInteger
+    assertExpr.from("""MAP["BOOLEAN"]""") shouldHaveInferredType CTBoolean
+    assertExpr.from("""MAP["baz"]""") shouldHaveInferredType CTVoid
```

Comment:
It would probably be a little easier to read if we used normal double-quotes on the outside (for Scala) and single-quotes (apostrophes) on the inside (for Cypher):

```suggestion
    assertExpr.from("MAP['baz']") shouldHaveInferredType CTVoid
```